packages/node_modules/@webex/plugin-logger/test/unit/spec/logger.js

```diff
@@ -388,12 +388,11 @@ describe('plugin-logger', () => {
     it('redact emails', () => {
       webex.config.logger.level = 'trace';
 
-      // This behaviour does not seem correct
```

Comment:
I like how the original developer knew something was off with the old RegEx but code still went through... 😭